.fixtures.yml

```diff
@@ -2,9 +2,7 @@ fixtures:
   repositories:
     sshkeys:
       repo: 'git://github.com/puppetlabs/puppetlabs-sshkeys_core.git'
-      ref: '1.0.1'
```

Comment:
The ref needs to be there so that we are testing against a known working version. These can change, but something needs to be there.

Reply:
Better like this?

Reply:
Did it not work with the versions that were already there? If so, suggest bumping to the earliest version that does work instead of what could be a big upgrade for some.

Reply:
We are only talking about the fixtures, which are the modules used in the tests, right?

Reply:
yes, that verifies the minimum bound in the metadata is correct.

Reply:
yes, that verifies the minimum bound in the metadata is correct.

Reply:
How do we verify the upper bound?

Reply:
The upper bound is whatever is current, which could fail. Check out https://github.com/sensu/sensu-puppet/blob/master/.fixtures-latest.yml

Reply:
Because of semantic versioning. When the major number changes it can break things. Check out https://semver.org/

Reply:
Okay, I'm familiar with semver. I'm mostly wondering, how do we know things don't break if we don't test against the most recent (supported) release of the dependencies? Assuming every other major and maybe even minor release was tested with no issue at some point because it was the most recent at that time, and potential issues were fixed when they came up (or the version limits were adjusted to reflect incompatibility). I know, probably out of scope of this PR, but it is the problem I'm facing right now updating many modules in our dependency graph, and I'm wondering why so many limits are set when there is no clear reason for it...
I realize of course there are some moments of backwards incompatibility (eg the systemd module removed the reload target), but that should mean a temporary upper limit with a clear reason, until that incompatibility is fixed... Anyway, preparing for the new work day, I'll fix the fixtures in an hour or two! ❤️